### PR TITLE
PLATUI-544: Remove partial mechanism

### PR DESCRIPTION
**Pull request by matthewmascord**
Update the GTM tag to be the production container
Turn off language toggle to facilitate deployment to production

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -69,6 +69,12 @@ microservice {
   }
 
   services {
+    tracking-consent-frontend  {
+      protocol = "http"
+      host = "localhost"
+      port = 12345
+      path = "/tracking-consent/tracking.js"
```
Comment (JoPintoPaul) [inline conf/application.conf:76]: Maybe more descriptive key name, like tracking-script-path?

File: app/uk/gov/hmrc/trackingconsentfrontend/config/AppConfig.scala [hunk 2]
```diff
@@ -22,7 +22,14 @@ import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
...
 class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig) {
-  val cookieBannerAssetsPrefix: String = config.getOptional[String]("cookie-banner.assets-prefix").getOrElse("")
+  private def loadConfig(key: String): String =
+    config
+      .getOptional[String](key)
+      .getOrElse(throw new Exception(s"Missing configuration key: $key"))
+
   val footerLinkItems: Seq[String] = config.getOptional[Seq[String]]("footerLinkItems").getOrElse(Seq())
+  val trackingConsentPath: String = loadConfig("microservice.services.tracking-consent-frontend.path")
+  val trackingConsentBaseUrl: String = servicesConfig.baseUrl("tracking-consent-frontend")
```
Comment (JoPintoPaul) [inline app/uk/gov/hmrc/trackingconsentfrontend/config/AppConfig.scala:32]: If you use servicesConfig.getString, you can remove the loadConfig function.

File: conf/app.routes [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ->         /govuk-frontend          govuk.Routes
 ->         /hmrc-frontend           hmrcfrontend.Routes
-GET        /consent-head            uk.gov.hmrc.trackingconsentfrontend.controllers.ConsentPartialController.head
+GET        /tracking.js             controllers.Assets.at(path="/public", file="servicePage.js")
```
Comment (JoPintoPaul) [inline conf/app.routes:5]: Should the asset file name match the URL?

File: conf/application.conf (hunk 1 above)
Comment (matthewmascord) [reply conf/application.conf:76]: What about script-path?

File: app/uk/gov/hmrc/trackingconsentfrontend/config/AppConfig.scala (hunk 2 above)
Comment (matthewmascord) [reply app/uk/gov/hmrc/trackingconsentfrontend/config/AppConfig.scala:32]: Perfect, that's what I was looking for.

File: conf/application.conf (hunk 1 above)
Comment (JoPintoPaul) [reply conf/application.conf:76]: I think that's better, yes.

File: conf/app.routes (hunk 3 above)
Comment (matthewmascord) [reply conf/app.routes:5]: It probably should really.

File: conf/application.conf [hunk 4]
```diff
@@ -93,4 +93,8 @@ auditing {
 }
 
 footerLinkItems = ["cookies", "privacy", "termsConditions", "govukHelp"]
-features.welsh-language-support = true
+features.welsh-language-support = false
+tracking-consent-frontend {
+  script-path = "/tracking-consent/tracking.js"
+  tracking-consent-frontend.assets-prefix = "http://localhost:12345"
```
Comment (JoPintoPaul) [inline conf/application.conf:99]: Should this key should be just: 
assets-prefix
not 
tracking-consent-frontend.assets-prefix

File: app/uk/gov/hmrc/trackingconsentfrontend/config/AppConfig.scala [hunk 5]
```diff
@@ -22,7 +22,15 @@ import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
 @Singleton
 class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig) {
-  val cookieBannerAssetsPrefix: String = config.getOptional[String]("cookie-banner.assets-prefix").getOrElse("")
+  private def loadConfig(key: String): String =
```
Comment (JoPintoPaul) [inline app/uk/gov/hmrc/trackingconsentfrontend/config/AppConfig.scala:25]: I think we discussed that loadConfig could be be deleted in favour of servicesConfig.getString(

**Pull request merged**